### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sxyazi**
Fix the `[[plugin]]` (should be `[plugin]`), and add some dependencies.

### CODE REVIEW

**Comment by GOWxx** [comment]
If you install pdftoppm using brew, there will be a warning:
```bash
Warning: No available formula with the name "pdftoppm". Did you mean pdftoipe?
```
A search reveals that pdftoppm is provided by the poppler project.

So, the actual dependency you need to install is:

```bash
brew install bat exiftool poppler ffmpeg
```

File: README.md [hunk 1]
```diff
@@ -13,12 +13,19 @@ git clone https://github.com/Urie96/preview.yazi.git ~/.config/yazi/plugins/prev
 # Windows is unsupported
 ```
 
+And make sure you have installed the following dependencies and they can be found in the `PATH`:
+
+- bat
+- exiftool
+- pdftoppm
+- ffmpeg
```
Comment (GOWxx) [inline README.md:25]:
```diff
- - pdftoppm
+ - poppler
```

**Review by GOWxx — COMMENTED** [review]
If you install pdftoppm using brew, there will be a warning:
```bash
Warning: No available formula with the name "pdftoppm". Did you mean pdftoipe?
```
A search reveals that pdftoppm is provided by the poppler project.

So, the actual dependency you need to install is:

```bash
brew install bat exiftool poppler ffmpeg
```

File: README.md (hunk 1 above)
Comment (sxyazi) [reply README.md:25]: Nice catch, thanks!

**Pull request merged**